packages/core/useUserMedia/demo.vue

```diff
@@ -20,6 +20,13 @@ watchEffect(() => {
   if (video.value)
     video.value.srcObject = stream.value!
 })
+
+watch(() => currentCamera.value, () => {
+  enabled.value = false
+  setTimeout(() => {
+    enabled.value = true
+  }, 100)
+})
```

Comment:
maybe you could briefly explain why this is necessary. Unfortunately, I don't have a second camera at the moment. 

And shouldn't this be controlled by the autoSwitch of useUserMedia? :)